Common/FeatureFlags.swift

```diff
@@ -89,6 +90,13 @@ struct FeatureFlagConfiguration: Decodable {
         #else
         self.includeServicesInSettingsEnabled = true
         #endif
+        
+        // Swift compiler config is inverse, since the default state is enabled.
+        #if MOCK_THERAPY_SETTINGS_ENABLED
+        self.mockTherapySettingsEnabled = false
+        #else
+        self.mockTherapySettingsEnabled = true
+        #endif
```

Comment:
are these now around the wrong way?

Reply:
The flags are super weird; @rickpasetto could you comment?

Reply:
Fixed